.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: scala
 sudo: false
-matrix:
```

Comment:
why is this change in here? other than the compile step being before the test step whats the gain/reasoning?